### PR TITLE
Include flag for keyword verification in application configuration.

### PR DESCRIPTION
**Pull request by olmidy**
## Purpose
Configuration for the bot connector sets properties for keyword verification according to configuration settings.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

*  Change app config settings
*  Also attempt without specifying "UseKeywordVerification" flag in app configuration.

## What to Check
Verify that the following are valid
*  Any provided tests should run appropriately.

### CODE REVIEW

File: clients/csharp-dotnet-core/voice-assistant-test/tool/AppSettings.cs [hunk 1]
```diff
@@ -119,6 +119,11 @@ internal class AppSettings
         /// </summary>
         public string KeywordRecognitionModel { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to enable keyword verification in the SDK.
```
Comment (dargilco) [inline clients/csharp-dotnet-core/voice-assistant-test/tool/AppSettings.cs:123]: From this sentence it sounds like KWV is done in the SDK. I would reword it such that it's clear it's done in the cloud.

File: clients/csharp-dotnet-core/voice-assistant-test/README.md [hunk 2]
```diff
@@ -105,6 +105,9 @@ Here is the full list:
 
 <font color="red">TODO: Make the above relative path to InputFolder?</font>
 
+>#### UseKeywordVerification
```
Comment (dargilco) [inline clients/csharp-dotnet-core/voice-assistant-test/README.md:108]: Looking at other boolean properties, looks like you can simply call it "KeywordVerification" (no need for the "Use"). Or "KeywordVerificationEnabled"

Comment (chschrae) [reply clients/csharp-dotnet-core/voice-assistant-test/README.md:108]: I like KeywordVerificationEnabled :)

File: clients/csharp-dotnet-core/voice-assistant-test/docs/json-templates/app-config-template.json [hunk 3]
```diff
@@ -30,6 +30,7 @@
   "SpeechSDKLogEnabled": false,
   "Timeout":  5000,
   "KeywordRecognitionModel" : "",
+  "UseKeywordVerification": true,
```
Comment (chschrae) [inline clients/csharp-dotnet-core/voice-assistant-test/docs/json-templates/app-config-template.json:33]: I think this has to change to fit the new name :)

Comment (chschrae) [inline clients/csharp-dotnet-core/voice-assistant-test/docs/json-templates/app-config-template.json:33]: I think this has to change for the new name

**Review by chschrae — CHANGES_REQUESTED** [review]

**Review by chschrae — APPROVED** [review]
